.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 
  # all intellij files
-.idea/
\ No newline at end of file
+.idea/
+.svg
```

Comment:
I'm not sure we want to not allow _any_ svg files into git?

Reply:
Sorry, I've forgot to remove it.